packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -46,69 +46,154 @@ import 'ticker_provider.dart';
 /// [OverscrollIndicatorNotification.paintOffset] to the
 /// notification, or use a [NestedScrollView].
 ///
-/// {@tool dartpad --template=stateless_widget_scaffold}
+/// {@tool dartpad --template=freeform}
 ///
 /// This example demonstrates how to use a [NotificationListener] to manipulate
 /// the placement of a [GlowingOverscrollIndicator] when building a
 /// [CustomScrollView]. Drag the scrollable to see the bounds of the overscroll
 /// indicator.
 ///
 /// ```dart
-/// Widget build(BuildContext context) {
-///   final double leadingPaintOffset = MediaQuery.of(context).padding.top + AppBar().preferredSize.height;
-///   return NotificationListener<OverscrollIndicatorNotification>(
-///     onNotification: (OverscrollIndicatorNotification notification) {
-///       if (notification.leading) {
-///         notification.paintOffset = leadingPaintOffset;
-///       }
-///       return false;
-///     },
-///     child: CustomScrollView(
-///       slivers: <Widget>[
-///         const SliverAppBar(title: Text('Custom PaintOffset')),
-///         SliverToBoxAdapter(
-///           child: Container(
-///             color: Colors.amberAccent,
-///             height: 100,
-///             child: const Center(child: Text('Glow all day!')),
+/// import 'package:flutter/gestures.dart';
+/// import 'package:flutter/material.dart';
+///
+/// // Override the [default drag behavior](https://flutter.dev/docs/release/breaking-changes/default-scroll-behavior-drag)
+/// // to demonstrate GlowingOverscrollIndicator on desktop.
+/// class MyCustomScrollBehavior extends MaterialScrollBehavior {
+///   @override
+///   Set<PointerDeviceKind> get dragDevices => {
+///     PointerDeviceKind.touch,
+///     PointerDeviceKind.mouse,
+///   };
+/// }
+///
+/// void main() => runApp(const MyApp());
+/// // This is the main application widget.
+/// class MyApp extends StatelessWidget {
+///   const MyApp({Key? key}) : super(key: key);
+///
+///   static const String _title = 'Flutter Code Sample';
```

Comment:
The title and this constant can be left out, since they don't really add to the sample.